content/updates/2021-10/1-TableAttributeSU.md

```diff
@@ -3,10 +3,11 @@ pim_announcement_img: ./img/TableAttribute-table.png
 pim_announcement_alt_img: "Table Attribute"
 pim_announcement_audience:
 - EE
+- GE
 ---
 
 # Manage your multidimensional data in Akeneo PIM
```

Comment:
Is this just an update to reflect that it's available in GE? Because Table Attributes was already included in the October updates

Reply:
This is to make sure it'll be visible for both, EE and GE users